websocket/connection.go

```diff
@@ -125,20 +125,20 @@ func NewDurableConnection(target string, messageChan chan []byte, logger *zap.Su
 		for {
 			select {
 			default:
-				logger.Infof("Connecting to %q", target)
+				logger.Infof("Connecting to %s", target)
```

Comment:
This seems spammy to have 3 of these logs in success cases. Convert one or two into `.Debugf`?

Reply:
Downgraded one that seemed redundant.